README.md

```diff
@@ -1 +1,26 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+>   Hello Microverse project aims at testing the micronauts ability to install, configure and use linters.
+    it also tests their ability to document professionally their repositories and proper use if their githubworkflows.
+
+
+## Built With
+
+- HTML
+- CSS
+- Linters
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/rugwizangoga)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/iradukunda-pacific-rugwizangoga)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
```

Comment:
- [ ] Kindly add a link for issue page like this  `Feel free to check the [issues page](../../issues/).`